src/command_modules/azure-cli-backup/setup.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup
+
+VERSION = "0.0.1b1+dev"
+
+# The full list of classifiers is available at
+# https://pypi.python.org/pypi?%3Aaction=list_classifiers
+CLASSIFIERS = [
+    'Development Status :: 4 - Beta',
+    'Intended Audience :: Developers',
+    'Intended Audience :: System Administrators',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 2',
+    'Programming Language :: Python :: 2.7',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.4',
+    'Programming Language :: Python :: 3.5',
+    'Programming Language :: Python :: 3.6',
+]
+
+DEPENDENCIES = [
+    'azure-mgmt-recoveryservices==0.1.0',
+    'azure-mgmt-recoveryservicesbackup==0.1.0',
+    'azure-cli-core',
+    'azure-cli-vm'
+]
+
+with open('README.rst', 'r', encoding='utf-8') as f:
+    README = f.read()
+with open('HISTORY.rst', 'r', encoding='utf-8') as f:
+    HISTORY = f.read()
+
+setup(
+    name='azure-cli-backup',
+    version=VERSION,
+    description='Microsoft Azure Command-Line Tools Recovery Services Command Module',
+    long_description=README + '\n\n' + HISTORY,
+    license='MIT',
+    author='Microsoft Corporation',
+    author_email='azpycli@microsoft.com',
+    url='https://github.com/Azure/azure-cli',
+    classifiers=CLASSIFIERS,
+    namespace_packages = [
+        'azure',
+        'azure.cli',
+        'azure.cli.command_modules',
```

Comment:
Move these to the `packages` section below.
See other command modules for examples.